navigator/frontend/src/ui-core/src/Button/index.tsx

```diff
@@ -103,18 +103,20 @@ export interface Props {
   theme?: ThemeInterface;
 }
 
-const Button = ({ onClick, disabled, className, children, autoFocus, tabIndex }: Props) => (
+const Button = React.forwardRef<HTMLButtonElement>(
```

Comment:
This was actually broken before since SDK 1.3.0 afaict and resulted in the choice popup you get when you click on the `gear` icon on the contracts screen.